### PR TITLE
New location icon and support for quantity edit

### PR DESCRIPTION
**Pull request by constanzauanini**
Screen Shot 2022-09-12 at 16 02 44
Screen Shot 2022-09-12 at 16 11 41
Screen Shot 2022-09-12 at 16 11 49

### CODE REVIEW

File: src/components/accession2/CalculatorModal.tsx
Comment (karthikbtf) [inline src/components/accession2/CalculatorModal.tsx:107]: in my most recent PR I created a new dropdown utility, it will be easier to use for later.

File: src/components/accession2/CalculatorModal.tsx
Comment (karthikbtf) [inline src/components/accession2/CalculatorModal.tsx:20]: call it `onPrevious`? this component shouldn't know whether previous ui was a modal or something else.

File: src/components/accession2/Accession2View.tsx
Comment (karthikbtf) [inline src/components/accession2/Accession2View.tsx:211]: can we give this cursor: pointer style? it has a text pointer for me

File: src/components/accession2/QuantityModal.tsx
Comment (karthikbtf) [inline src/components/accession2/QuantityModal.tsx:109]: looks like this for me 
Terraware App 2022-09-12 13-14-59

File: src/components/accession2/CalculatorModal.tsx
Comment (karthikbtf) [inline src/components/accession2/CalculatorModal.tsx:29]: this updates the accession in the database,, is that what we want? i thought this modal just does some calculation for us.

File: package.json
Comment (karthikbtf) [inline package.json:19]: just a request - can we bump this to 20 just to pick up latest change.. and run yarn? otherwise i can send a separate PR but want to avoid a conflict.

File: src/components/accession2/CalculatorModal.tsx
Comment (karthikbtf) [reply src/components/accession2/CalculatorModal.tsx:107]: i merged my SelectValue component if we want to use that

File: package.json
Comment (constanzauanini) [reply package.json:19]: Sure!

File: src/components/accession2/CalculatorModal.tsx
Comment (constanzauanini) [reply src/components/accession2/CalculatorModal.tsx:29]: Yes, we need to get the estimated seed count value, and for doing that we have to save values and get the value from the api, but now that you mention I forgot to reset it back to original values if you press cancel. Will do that

File: src/components/accession2/QuantityModal.tsx
Comment (constanzauanini) [reply src/components/accession2/QuantityModal.tsx:109]: Oh, my bad. I didn't push last changes :(

File: src/components/accession2/CalculatorModal.tsx
Comment (karthikbtf) [reply src/components/accession2/CalculatorModal.tsx:29]: Ok, i think we can ask Steve for a preview api .. if user does not click cancel and closes browser we have modified data.

File: src/components/accession2/CalculatorModal.tsx
Comment (constanzauanini) [reply src/components/accession2/CalculatorModal.tsx:107]: Yes, will do!

File: src/components/accession2/CalculatorModal.tsx
Comment (constanzauanini) [reply src/components/accession2/CalculatorModal.tsx:29]: I was going to save the original values and restore them (save the accession again with those values) before onClose (cancel or clicking outside call the onClose method). But it sounds cleaner to have an api for getting that value.

Comment (karthikbtf) [reply src/components/accession2/CalculatorModal.tsx:29]: @constanzauanini actually, you can use the 'simulate: true' option .. the API will return the values but won't update data on disk. We just need to pass `?simulate=true` param with the API

Comment (constanzauanini) [reply src/components/accession2/CalculatorModal.tsx:29]: Oh, that's true! I forgot about that! Excellent! Thank you

File: src/components/accession2/CalculatorModal.tsx
Comment (karthikbtf) [reply src/components/accession2/CalculatorModal.tsx]: Replace these, I will rename to Dropdown later as part of my PR https://github.com/terraware/terraware-web/pull/496.. which I won't merge until after this PR.

**Review by karthikbtf — APPROVED** [review]
Looks good for now, if you can replace SelectT with SelectValue before merging that will be easier for me to replace.

**Review by karthikbtf — APPROVED** [review]
🎉

**Pull request merged**
